clappr/clappr-ios
# Create Core and Container Factories

# Goal

Refactor to improve *Clappr's Architecture* scalability to test new Playbacks. Where we:

* Changed the way how we instantiate a `Core` and `Container` in `Player`.
* Changed the playbacks that exist in Loader from `[Plugin.Type]` to `[Playback.Type]`

# How to test

- Run all tests
- Check if the video loads properly
- Use `configure` to reload a video
- Use `load` to reload a video
